docs/source/algorithms/PDCalibration-v1.rst

```diff
@@ -43,7 +43,7 @@ When not specified using the ``MaskWorkspace`` parameter, the default name for t
 
 The resulting calibration table can be saved with
 :ref:`algm-SaveDiffCal`, loaded with :ref:`algm-LoadDiffCal` and
-applied to a workspace with :ref:`algm-AlignDetectors`. There are also
+applied to a workspace with :ref:`algm-ConvertUnits`. There are also
```

Comment:
I think we should also mention `ApplyDiffCal` - which actually uses the calibration table and updates the diffractometer constants in the `UnitParams` map of the workspace